docs/Troubleshooting.md

```diff
@@ -289,4 +290,33 @@ Or
 sudo sh ./onboard_agent.sh --purge
 ```
 
-You can continue re-onboarding after using the `--purge` option
\ No newline at end of file
+You can continue re-onboarding after using the `--purge` option
+
+
+### OMS extension on Azure portal is marked as failed state: Provisioning failed
```

Comment:
We recommend azure customer use OMS Linux VM extension to install omsagent and not command line tool. Please also review existing troubleshooting on omsagent extension on possible errors are documented here: https://github.com/Azure/azure-linux-extensions/tree/master/OmsAgent
And move this content to be in extension repo instead of omsagent repo